keras_core/layers/merging/concatenate.py

```diff
@@ -69,7 +69,8 @@ def build(self, input_shape):
                 if axis != concat_axis and axis_value == 1:
                     del reduced_inputs_shapes[i][axis]
 
-            del reduced_inputs_shapes[i][self.axis]
+            if len(reduced_inputs_shapes[i]) > self.axis:
```

Comment:
Why did this come up?

Reply:
In the example, in Inception V3, some axes were squeezed out, and thus, there were error messages showing out-of-range while accessing the `reduced_inputs_shapes[i][self.axis]`. Alternatives I could think of:
1. Don’t squeeze
2. When squeezing, also decrement `self.axis`